### DISCUSSION TITLE
How to force auto-update of PWA?

### DISCUSSION THREAD

**Discussion by jonmor51** [quasarframework/quasar#13176] [CLI - PWA mode] [answered]
I've developed a Quasar web application which is running as a PWA. When I upload a new version to the server, people are not automatically seeing the new code, they're having to explicitly force a page refresh. Is there some way of configuring things so that updated code is automatically recognised and loaded?

**Comment by metalsadman** [comment, marked as answer]
do it in your `src-pwa/register-service-worker.js` `updated` handler.
```js
register(process.env.SERVICE_WORKER_FILE, {
  ...
  updated (/* registration */) {
    location.reload()
  }
  ...
}
```
